src/I18n/composer.json

```diff
@@ -1,26 +1,48 @@
 {
     "name": "cakephp/i18n",
     "description": "CakePHP Internationalization library with support for messages translation and dates and numbers localization",
+    "type": "library",
+    "keywords": [
+        "cakephp",
+        "i18n",
+        "internationalisation",
+        "internationalization",
+        "localisation",
```

Comment:
We should only use the US english (norm) term IMO - as we do with behavior and color etc.